README.md

```diff
@@ -532,8 +532,8 @@ Open an [issue](https://github.com/solarkennedy/puppet-consul/issues) or
 
 ## Transfer Notice
 
-This plugin was originally authored by [KyleAnderson](http://github.com/KyleAnderson).
+This module was originally authored by [KyleAnderson](http://github.com/KyleAnderson).
```

Comment:
That's not the correct username either. See https://github.com/voxpupuli/puppet-consul/pull/593#discussion_r1011871459

Reply:
I'm not sure how I initially created it, but that's at least the forge handle (which is different to GitHub). Updated it in https://github.com/voxpupuli/puppet-consul/pull/620